packages/financial-templates-lib/src/logger/SlackTransport.js

```diff
@@ -163,6 +163,71 @@ function slackFormatter(info) {
   }
 }
 
+class SlackHook extends Transport {
```

Comment:
Can you briefly explain what this is replacing? I'm assuming we were previously using a built-in slack transport, but we had a custom slack formatter, but now we have both a custom formatter and custom transport?

Reply:
Previously we were using the [winston-slack-transport](https://github.com/TheAppleFreak/winston-slack-webhook-transport#readme) but this had limitations on the size of slack messages that could get sent over the transport. The implementation would init a new instance of their [SlackTransport](https://github.com/TheAppleFreak/winston-slack-webhook-transport/blob/a6d3bc82692cd7737593de1cd12b2ad44dc4f703/slackHook.js#L6). 

After this PR we implement a more similar pattern to what we do in the [`PagerDudty`](https://github.com/UMAprotocol/protocol/blob/ad3bac05ef38a17ebf2be370e4e61a1452f5291e/packages/financial-templates-lib/src/logger/PagerDutyTransport.js#L10) transport which is our own custom transport. The standard pattern for these transports is to extend the Transport class from Winston and then implement a `log` function. We could merge the formatter into this transport but it felt cleaner to leave it as two separate functions. 

I might take another stab at refactoring all the transports in this directory to be more consistent in how they are exported but for now, I think it's sufficient.